src/coreclr/tests/src/CLRTest.CrossGen.targets

```diff
@@ -83,8 +83,7 @@ if [ ! -z ${RunCrossGen2+x} ]%3B then
     rm $__ResponseFile
 
     __Command=$_DebuggerFullPath
-    __Command+=" $CORE_ROOT/corerun"
-    __Command+=" $CORE_ROOT/crossgen2/crossgen2.dll"
+    __Command+=" $CORE_ROOT/crossgen2/crossgen2.exe"
```

Comment:
I think you have these the other way around - this is the Linux version that shouldn't have the exe extension and below is the Windows version that *may* have it. But I may be completely off :-).

Reply:
Yes, it will work better that way round - thanks!